src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/OptionsTests.cs

```diff
@@ -1036,6 +1036,31 @@ public static void JsonSerializerOptions_Default_IsReadOnly()
             Assert.Same(resolver, optionsSingleton.TypeInfoResolver);
         }
 
+        [Fact]
+        public static void JsonSerializerOptions_Web_MatchesConstructorWithJsonSerializerDefaults()
```

Comment:
Please also add a test that attempts to serialize a value using the new singleton. I don't believe this would work with the current implementation.